mev_inspect/classifiers/specs/aave.py

```diff
@@ -74,13 +79,32 @@ def get_transfer(trace: DecodedCallTrace) -> Transfer:
         )
 
 
-AAVE_SPEC = ClassifierSpec(
-    abi_name="AaveLendingPool",
-    protocol=Protocol.aave,
-    classifiers={
-        "liquidationCall(address,address,address,uint256,bool)": AaveLiquidationClassifier,
-    },
-)
+AAVE_SPEC = [
+    ClassifierSpec(
+        abi_name="AaveLendingPool",
+        protocol=Protocol.aave,
+        classifiers={
+            "liquidationCall(address,address,address,uint256,bool)": AaveLiquidationClassifier,
+        },
+        valid_contract_addresses=["0x398eC7346DcD622eDc5ae82352F02bE94C62d119".lower()],
+    ),
+    ClassifierSpec(
+        abi_name="AaveLendingPoolV2",
+        protocol=Protocol.aave_v2,
+        classifiers={
+            "liquidationCall(address,address,address,uint256,bool)": AaveLiquidationClassifier,
+        },
+        valid_contract_addresses=["0x7d2768de32b0b80b7a3454c06bdac94a69ddc7a9"],
```

Comment:
how come this one doesn't need `lower()`?
shouldn't we apply the operator later?

Reply:
done in [2cec7fe](https://github.com/defi-wonderland/mev-inspect-py/pull/4/commits/2cec7feffd3adf3ba7ec5710354b8ed678057ce5)